pom.xml

```diff
@@ -99,7 +99,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>animal-sniffer-maven-plugin</artifactId>
-        <version>1.15</version>
+        <version>1.17-SNAPSHOT</version>
```

Comment:
You must deploy a timestamped snapshot for CI to proceed. FTR my `~/.bashrc`:

```bash
alias deploysnapshot='mvn clean install source:jar deploy:deploy -DaltDeploymentRepository=maven.jenkins-ci.org::default::https://repo.jenkins-ci.org/snapshots/ -DskipTests'
```

Also please include a `TODO` comment in the POM linking to the upstream PR so we can see when it is merged and released.